2fa-api/handlers/login.js

```diff
@@ -0,0 +1,45 @@
+const db = require("../db");
+const twilio = require("../twilio");
+
+const twilioClient = twilio.twilioClient;
+
+const login = function (req, res) {
+  try {
+    const { email } = req.body;
+    const { password } = req.body;
+
+    db.connection.query(
+      `SELECT id, phone from Users WHERE email='${email}' and password='${password}'`,
+      (error, results, _) => {
+        if (error) {
+          console.log(error);
+          return res.status(500).json({ error });
+        }
+
+        if (!results.length) {
```

Comment:
what if `results > 1`?

Reply:
A success message will be returned, but I should ensure unique email field in the DB and this statement would only return results <=1

Reply:
it should not be successful... if results > 1, something fucked up on your end and you shouldn't return anything because you can't determine who is who (or if they're the same person)

Reply:
fixed